TestProject/Pages/Certifications.cs

```diff
@@ -0,0 +1,150 @@
+﻿using MarsQA_Nunit.Models;
+using MarsQA_Nunit.Utilities;
+using OpenQA.Selenium;
+
+namespace MarsQA_Nunit.Pages
+{
+    public class Certifications
+    {
+        private readonly By certificationTabLocator = By.XPath("//*[@id='account-profile-section']/div/section[2]/div/div/div/div[3]/form/div[1]/a[4]");
+        IWebElement certificationTab;
+        private readonly By addNewBtnLocator = By.XPath("//*[@id='account-profile-section']/div/section[2]/div/div/div/div[3]/form/div[5]/div[1]/div[2]/div/table/thead/tr/th[4]/div");
+        IWebElement addNewBtn;
+        private readonly By yearLocator = By.XPath("//*[@id='account-profile-section']/div/section[2]/div/div/div/div[3]/form/div[5]/div[1]/div[2]/div/div/div[2]/div[2]/select");
+        IWebElement year;
+        private readonly By addBtnLocator = By.XPath("//*[@id='account-profile-section']/div/section[2]/div/div/div/div[3]/form/div[5]/div[1]/div[2]/div/div/div[3]/input[1]");
+        IWebElement addBtn;
+        private readonly By editIconLocator = By.XPath("//*[@id='account-profile-section']/div/section[2]/div/div/div/div[3]/form/div[5]/div[1]/div[2]/div/table/tbody[last()]/tr/td[4]/span[1]/i");
+        IWebElement editIcon;
+        private readonly By updateBtnLocator = By.XPath("//*[@id='account-profile-section']/div/section[2]/div/div/div/div[3]/form/div[5]/div[1]/div[2]/div/table/tbody/tr/td/div/span/input[1]");
+        IWebElement update;
+        private readonly By certificateFromLocator = By.XPath("//*[@id='account-profile-section']/div/section[2]/div/div/div/div[3]/form/div[5]/div[1]/div[2]/div/table/tbody/tr/td/div/div/div[2]/input");
+        IWebElement certificateFromToBeUpdated;
+        private readonly By certificateLocator = By.XPath("//*[@id='account-profile-section']/div/section[2]/div/div/div/div[3]/form/div[5]/div[1]/div[2]/div/table/tbody/tr/td/div/div/div[1]/input");
+        IWebElement certificate;
+        private readonly By yearToBeLocator=By.XPath("//*[@id='account-profile-section']/div/section[2]/div/div/div/div[3]/form/div[5]/div[1]/div[2]/div/table/tbody/tr/td/div/div/div[3]/select");
+        IWebElement yearToBeUpdated;
+        private readonly By deleteIconLocator = By.XPath("//*[@id='account-profile-section']/div/section[2]/div/div/div/div[3]/form/div[5]/div[1]/div[2]/div/table/tbody[last()]/tr/td[4]/span[2]/i");
+        IWebElement deleteIcon;
+
+        //Method to create new certification record
+        public void CreateNewCertifications(IWebDriver driver,CreateCertification record)
```

Comment:
Please do not pass the driver In action methods (methods which in pages doing things like create, delete etc ) instead get the pages to extend to common driver where you will create a common driver class which will initialise the web driver 

public class CommonDriver
    {
        public static IWebDriver driver;

        public void initialise()
        {
            // create driver
            driver = new ChromeDriver();

        }

    }

Reply:
please fix this everywhere